commands/group.go

```diff
@@ -308,3 +329,16 @@ func teamGroupListCmdF(c client.Client, cmd *cobra.Command, args []string) error
 
 	return nil
 }
+
+func userGroupRestoreCmdF(c client.Client, cmd *cobra.Command, args []string) error {
+	groupID := args[0]
+	_, resp, err := c.RestoreGroup(groupID, "")
+
+	if err != nil {
+		return err
+	}
+	if resp.StatusCode == http.StatusOK {
+		printer.Print("Group successfully restored with ID: " + groupID)
+	}
+	return nil
```

Comment:
minor styling suggestion:
```suggestion
	if err != nil {
		return err
	}

	if resp.StatusCode == http.StatusOK {
		printer.Print("Group successfully restored with ID: " + groupID)
	}

	return nil
```

Reply:
Additionally, is there any case that server returns `http.StatusOK` when there is no `error`?

Reply:
Thanks for pointing it out, https://github.com/mattermost/mattermost-server/pull/20369/files#diff-060eb667249b07159dcd89b520fa51777abf5bd355241d2e5c7e6bb58a93d367R1178 it only returns when RestoreGroup succeeds.